triage/templates/triage/wizard-step-base.html

```diff
@@ -11,7 +11,7 @@
     <section class="questions {% block container_class %}{% endblock %}">
         <div class="container">
             <div class="question ">
-				<form action="" method="post" enctype="multipart/form-data">
+				<form action="{% url 'triage-wizard' %}" method="post" enctype="multipart/form-data">
```

Comment:
fun fact: before this change, the current url is used - so the browser could post to `/triage?result=true` - meaning when the user made their way back to the summary page having reviewed their answers...the summary would show their persisted answers, not their new answers.

I have added a test to check this no longer happens.